---a/net/minecraft/stats/AchievementList.java
+++b/net/minecraft/stats/AchievementList.java
@@ -1,20 +1,21 @@
 package net.minecraft.stats;
 
-import com.google.common.collect.Lists;
 import java.util.List;
+
 import net.minecraft.init.Blocks;
 import net.minecraft.init.Items;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.JsonSerializableSet;
 
-public class AchievementList
-{
+import com.google.common.collect.Lists;
+
+public class AchievementList {
+
     public static int minDisplayColumn;
     public static int minDisplayRow;
     public static int maxDisplayColumn;
     public static int maxDisplayRow;
     public static List achievementList = Lists.newArrayList();
-    public static Achievement openInventory = (new Achievement("achievement.openInventory", "openInventory", 0, 0, Items.book, (Achievement)null)).setIndependent().func_180788_c();
+    public static Achievement openInventory = (new Achievement("achievement.openInventory", "openInventory", 0, 0, Items.book, (Achievement) null)).setIndependent().func_180788_c();
     public static Achievement mineWood = (new Achievement("achievement.mineWood", "mineWood", 2, 1, Blocks.log, openInventory)).func_180788_c();
     public static Achievement buildWorkBench = (new Achievement("achievement.buildWorkBench", "buildWorkBench", 4, -1, Blocks.crafting_table, mineWood)).func_180788_c();
     public static Achievement buildPickaxe = (new Achievement("achievement.buildPickaxe", "buildPickaxe", 4, 2, Items.wooden_pickaxe, buildWorkBench)).func_180788_c();
@@ -50,5 +50,6 @@
     public static Achievement overpowered = (new Achievement("achievement.overpowered", "overpowered", 6, 4, Items.golden_apple, buildBetterPickaxe)).setSpecial().func_180788_c();
     private static final String __OBFID = "CL_00001467";
 
-    public static void init() {}
-}
+    public static void init() {
+    }
+}
